fill-dirt.lic

```diff
@@ -0,0 +1,31 @@
+custom_require.call(%w[common common-items drinfomon])
+
+class ForageDirt
+  include DRC
+  include DRCI
+
+  def initialize
+    @dirt_stacker = get_settings.dirt_stacker
+    fill_stacker unless bput("rummage #{@dirt_stacker}",
+                             'it is full of dirt',
+                             'it is almost full of dirt',
+                             'over three quarters full',
+                             'over half full',
+                             'over a quarter full',
+                             'has a little dirt',
+                             'out of dirt') == 'it is full of dirt'
+  end
+
+  def fill_stacker
+    loop do
+      forage?('dirt')
```

Comment:
Do you want to break if `forage?` fails?

Reply:
Good idea; done.